samples/CommunityToolkit.Maui.Sample/ViewModels/ViewControls/AvatarModel.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace CommunityToolkit.Maui.Sample.ViewModels.ViewControls;
+
+public class AvatarModel
+{
+	public Color? AvatarBackgroundColor { get; set; } = Colors.Black;
```

Comment:
Please make this Property read-only and remove nullability

```suggestion
	public Color AvatarBackgroundColor { get; init; } = Colors.Black;
```

Reply:
Fully understand and will do so.